influxdb3/src/commands/serve.rs

```diff
@@ -300,12 +300,17 @@ pub async fn command(config: Config) -> Result<()> {
         config.query_log_size,
     ));
 
+    let listener = TcpListener::bind(*config.http_bind_address)
```

Comment:
We have some tests that use a socket addr and I think they cause spurious failures. I'm glad we bind here and pass this in

Reply:
Yes! That is largely what motivated me to do this 😊